app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    command_name, source_file, destination_file = command_parts
```

Comment:
I would write a length check immediately after the split and than unpacking

Reply:
Resolved